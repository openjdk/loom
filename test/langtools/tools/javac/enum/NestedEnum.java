/*
 * @test /nodynamiccopyright/
 * @bug 5071831
 * @summary javac allows enum in an inner class for release >= 16
 * @author gafter
 *
 * @compile/fail/ref=NestedEnum.out -XDrawDiagnostics --release 15 NestedEnum.java
 * @compile NestedEnum.java
 */

class NestedEnum {
    class Inner {
        enum NotAllowedInNonStaticInner {}
    }
}
