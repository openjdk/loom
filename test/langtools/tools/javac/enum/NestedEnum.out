NestedEnum.java:13:9: compiler.err.static.declaration.not.allowed.in.inner.classes
1 error
